homeassistant/components/lcn/manifest.json

```diff
@@ -8,5 +8,5 @@
   "documentation": "https://www.home-assistant.io/integrations/lcn",
   "iot_class": "local_push",
   "loggers": ["pypck"],
-  "requirements": ["pypck==0.8.5", "lcn-frontend==0.2.4"]
+  "requirements": ["pypck==0.8.6", "lcn-frontend==0.2.5"]
```

Comment:
Can we bump in separate PRs?

Reply:
Yes, it's possible to only bump `pypck`. However, the positioning can only be configured once the subsequent frontend PR has also been merged. Until then, everything remains as it was from the user`s perspective.